src/widgets/tdarr/TdarrQueueTile.tsx

```diff
@@ -0,0 +1,285 @@
+              </Code>
+            )}
+            {workers.isError && (
+              <Code mt="sm" block>
+                {workers.error.message}
+              </Code>
+            )}
+            {queue.isError && (
+              <Code mt="sm" block>
+                {queue.error.message}
+              </Code>
+            )}
+          </List>
+        </Alert>
+      </Group>
+    );
+  }
+
+  if (!app) {
+    return (
+      <Stack
+        justify="center"
+        style={{
+          height: '100%',
+        }}
+      >
+        <Center style={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+          <Title order={3}>{t('noAppSelected')}</Title>
+        </Center>
+      </Stack>
+    );
+  }
+
+  const totalQueuePages = Math.ceil((queue.data?.totalCount || 1) / queuePageSize);
+
+  return (
+    <Stack
+      spacing="xs"
+      style={{
+        height: '100%',
+      }}
+    >
+      {view === 'workers' ? (
+        <WorkersPanel workers={workers.data} isLoading={workers.isLoading} />
+      ) : view === 'queue' ? (
+        <QueuePanel queue={queue.data} isLoading={queue.isLoading} />
+      ) : (
+        <StatisticsPanel statistics={statistics.data} isLoading={statistics.isLoading} />
+      )}
+      <Divider />
+      <Group spacing="xs">
+        <SegmentedControl
+          data={[
+            {
+              label: (
+                <Center>
+                  <IconCpu2 size={18} />
+                  <Text size="xs" ml={8}>
+                    {t('tabs.workers', { count1: workers.data?.length ?? '?' })}
+                  </Text>
+                </Center>
+              ),
+              value: 'workers',
+            },
+            {
+              label: (
+                <Center>
+                  <IconClipboardList size={18} />
+                  <Text size="xs" ml={8}>
+                    {t('tabs.queue', { count1: workers.data?.length ?? '?' })}
+                  </Text>
+                </Center>
+              ),
+              value: 'queue',
+            },
+            {
+              label: (
+                <Center>
+                  <IconReportAnalytics size={18} />
+                  <Text size="xs" ml={8}>
+                    {t('tabs.statistics', { count1: workers.data?.length ?? '?' })}
+                  </Text>
+                </Center>
+              ),
+              value: 'statistics',
+            },
+          ]}
+          value={view}
+          onChange={(value) => setView(viewSchema.parse(value))}
+          size="xs"
+        />
+        {view === 'queue' && !!queue.data && (
+          <>
+            <Pagination.Root total={totalQueuePages} value={page} onChange={setPage} size="sm">
+              <Group spacing={5} position="center">
+                <Pagination.First disabled={page === 1} />
+                <Pagination.Previous disabled={page === 1} />
+                <Pagination.Next disabled={page === totalQueuePages} />
```

Comment:
Not doing pagination for workers?

Reply:
Unfortunately I wasn't able to reverse engineer how the Tdarr React app gets the workers with pagination. I agree that this would be preferrable.

Reply:
That's where the MantineReactTable could help, even if the api doesn't provide pagination, it'll still put the data in pages.